### DISCUSSION TITLE
Opening Cypress...Passthrough is not supported, GL is disabled, ANGLE is

### DISCUSSION THREAD

**Discussion by dteneva** [cypress-io/cypress#21014] [Cypress Cloud]
I need help with this!
This is my first time using Cypress. 

This is what I did but I can't open it. 
Why?
![Без име](https://user-images.githubusercontent.com/101521740/162616190-4c184b24-3eed-4700-b347-f3dba59a797d.png)

node_modules/.bin/cypress open
✔  Verified Cypress! C:\Users\DESI\AppData\Local\Cypress\Cache\9.5.3\Cypress

Opening Cypress...

[11688:0410/142756.087:ERROR:gpu_init.cc(453)] Passthrough is not supported, GL is disabled, ANGLE is

**Comment by dteneva** [comment]
Please, I need help!!!

**Reply by jjhelguero** [reply]
What is the reason for the `cypress` and `cypress.exe` files?

**Reply by dteneva** [reply]
Well, I guess I tried to download and extract the Cypress on the same CypressAutomation folder.

**Comment by ghost** [comment]
I get the same error using the 9.6.0 cypress/included Docker image on Ubuntu. Tests run fine but the noise in the output is distracting.

For what it's worth, I also get that error when running the tests with Firefox.

**Comment by kiusi-qa** [comment]
Hello. having the same issue here.

**Comment by SreekanthPDas** [comment]
I am also getting the same with Cypress 10.0.3. Any resolution yet ? Please help

**Comment by antomer** [comment]
getting same error when running tests in `cypress/included:9.6.0`

**Comment by IT19237114** [comment]
I'm having the same error. Any help?

**Comment by ShilpaJingle** [comment]
i am finding the same issue. Can anyone please support
![image](https://user-images.githubusercontent.com/108930741/190649985-0dcb9cae-734e-41ec-8593-ff9bebec1eca.png)

**Comment by jovand93** [comment]
In my case it was multiple screens.
Anyone using an extended screen run npx cypress open, wait for the cypress icon to be visible inside taskbar. When you hower cypress icon with mouse, right click the popup window and click maximize.
Hope this helps.
